pages/policy.html

```diff
@@ -5,6 +5,22 @@
 ---
 
 <div id="section-full">
+    <div class="section-header">
+        <h3>Gameplay streaming, Let's Play content, and Monetization Statement</h3>
+    </div>
+    <div class="section-item">
+        <div class="content">
+            <p>The core game of OpenTTD is released under the GNU General Public License (GPL) version 2, as are the OpenGFX, OpenSFX, and OpenMSX basetset content packs. The GPL permits use of the software for any purpose including commercial purposes.</p>
+            <h3>General permission for the unmodified game</h3>
+            <p>It is the intention of the development teams behind OpenTTD and the open baseset content packs that any Let's Play, review, and other similar content, is permitted with no limitations. Monetization and commercial use of the content is permitted without limitations.</p>
+            <h3>Non-coverage of use of other baseset data</h3>
+            <p>Note that the above statement of general permission only applies to using the OpenTTD core game with the OpenGFX graphics baseset, and optionally the OpenSFX sound effects baseset, and optionally OpenMSX music baseset.</p>
```

Comment:
twice the `optionally` is a bit weird. Maybe just put OpenSFX / OpenMSX together?